plugins/include/nvault.inc

```diff
@@ -21,43 +21,128 @@
 	#pragma loadlib nvault
 #endif
 
-/* All timestamps are in UNIX epoch form. */
+/**
+ * All timestamps are in UNIX epoch form.
+ */
 
-/* Opens a vault by name (such as "myvault")
- * Returns a vault id, INVALID_HANDLE otherwise (-1)
+/**
+ * Opens a vault by name. Creates a vault if it doesn't exist yet.
+ *
+ * @param name			Name of the vault. The vault will be created in
+ * 						/amxmodx/data/vault directory.
```

Comment:
It will be created in the directory specified by `amxx_datadir` so this needs to be either less or more specific depending on your preference. (https://mxr.alliedmods.net/amxmodx/source/modules/nvault/amxxapi.cpp#49)